README.md

```diff
@@ -74,6 +74,16 @@ This following command generates an [Apache RAT](https://creadur.apache.org/rat/
 $ sbt ratCheck
 ```
 
+### sbt-scoverage
```

Comment:
Does it make sense to change the heading to something like "Test Coverage Report" or something? If someone doesn't know what "sbt-socverage" is, this heading isn't very useful. The content of the section goes on to explain and link to sbt-scoverage, so that's good for people that want to know more details.

Reply:
Will change, just kept it that way since I saw the previous section was Apache RAT.

Reply:
Fair, Apache RAT probably isn't a very heading name, I'll take the blame for that.